docs/pages/setup/admin/labels.mdx

```diff
@@ -3,29 +3,36 @@ title: Labels
 description: Labeling Nodes and Applications
 ---
 
-This guide explains how to label nodes with Open Source, Enterprise Teleport,
-self-hosted or cloud editions.
+This guide explains how to label Teleport Nodes.
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Labeling Nodes and Applications
+## Labeling nodes and applications
```

Comment:
1. I thought we were capitalizing Node
2. Why mention nodes and apps here, given the title is just "how to label Teleport Nodes" and there are many other resources that also can be labeled (kube clusters, dbs, desktops, etc) What's special about applications that they get to be included here?

Reply:
This is how it was before I started editing the guide, and a full update was out of scope for that particular sprint. I've replaced "Nodes and applications" with variations on "Teleport resources".